Watchman.DomainModel/Users/Queries/Handlers/GetWarnEventsQueryHandler.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Watchman.Cqrs;
+using Watchman.Integrations.MongoDB;
+
+namespace Watchman.DomainModel.Users.Queries.Handlers
+{
+    public class GetWarnEventsQueryHandler : IQueryHandler<GetWarnEventsQuery, GetWarnEventsQueryResults>
+    {
+        private readonly ISessionFactory _sessionFactory;
+
+        public GetWarnEventsQueryHandler(ISessionFactory sessionFactory)
+        {
+            this._sessionFactory = sessionFactory;
+        }
+
+        public GetWarnEventsQueryResults Handle(GetWarnEventsQuery query)
+        {
+            using var session = this._sessionFactory.Create();
+
+            var warnEvents = session.Get<WarnEvent>().ToList();
+            List<WarnEvent> filteredEvents = new List<WarnEvent>();
+
+            if (query.ServerId == 0)
+            {
+                filteredEvents = warnEvents.Where(x => (x.Receiver.Id == query.UserId)).ToList();
+            }
+            else
+            {
+                filteredEvents = warnEvents.Where(x => (x.Receiver.Id == query.UserId) && (x.ServerId == query.ServerId)).ToList();
```

Comment:
po co te nawiasy?

Reply:
i czemu ToList skoro i tak zwracasz IEnumerable

Reply:
to nic innego niż dodatkowe obciążenie

Reply:
wydaje mi się że z nawiasami jest czytelniej

Reply:
no tak niezbyt
bo i tak && oddziela te warunki

Reply:
ok zmieniłem